javascript-practice/src/js/views/dashboardView.js

```diff
@@ -22,6 +40,19 @@ export default class DashboardView {
     this.addPasswordEl = document.getElementById('addPassword');
     this.addConfirmPasswordEl = document.getElementById('addConfirmPassword');
 
+    this.addUserErrorEls = {
+      addUserIdEl: document.querySelector('.add-user-error.add-user-id'),
```

Comment:
Id field should only generate automatically by mock API. Just show it, but not accept to edit or enter value